src/EditorFeatures/CSharpTest/CodeActions/InlineTemporary/InlineTemporaryTests.cs

```diff
@@ -3766,5 +3766,29 @@ public static void M(string s)
     }
 }");
         }
+
+        [WorkItem(4583, "https://github.com/dotnet/roslyn/issues/4583")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsInlineTemporary)]
+        public void DontParenthesizeInterpolatedString()
+        {
+            Test(
+            @"
+class C
+{
+    public void M()
+    {
+        var [|s1|] = $""hello"";
```

Comment:
Sure, though I don't think it's needed. This is simply the case where there's an interpolated string expression being inlined. Whether or not it has interpolations shouldn't matter. I'll add one for sanity though.